coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -74,3 +83,36 @@ def to_snakecase(string):
     return re.sub("[A-Z]",
                   lambda match: "_" + match.group(0).lower(),
                   string)
+
+
+def to_spacecase(string):
+   """
+   Converts the given string to space-case.
+
+   >>> to_spacecase('helloWorld')
+   'Hello World'
+   >>> to_spacecase('__Init__File__')
+   'Init File'
+   >>> to_spacecase('')
+   ''
+   >>> to_spacecase('Already Space Case')
+   'Already Space Case'
```

Comment:
@Asnelchristian what is the expected outpuot for a string like `ABCD`?

`A B C D` or `ABCD` itself?